### PR TITLE
mc38473 remote storage

### PR DESCRIPTION
**Pull request by hguthrie**
<DRAFT>

This PR cleans up the earlier draft work for remote storage adapter configurations.

Requires a Technical review before merging to the S3 integration branch.
This is not a final work.

### CODE REVIEW

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Create a private bucket for AWS S3.
+1. (_Optionally_) Generate access keys and secret keys.
+1. Configure Magento to use the private bucket.
+
+## Nginx configuration
+
+Nginx requires additional configuration to perform Authentication with the `proxy_pass` directive.
+
+We recommend using the `[ngx_aws_auth]` module.
```
Comment (shiftedreality) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md:25]: For some reason, the link is broken

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+```bash
+bin/magento remote-storage:enable aws-s3 region-name bucket-name [optional prefix] --access-key=<optional-access-key> --secret-key=<optional-secret-key>
+```
+
+The following table lists the parameters available for configuring the storage adapter.
+
+| Command line Parameter | Parameter name | Meaning | Default value |
+|--- |--- |--- |--- |
+| `remote-storage-adapter` | adapter | Name of adapter to use | none |
```
Comment (shiftedreality) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:40]: Please change to

```
| `remote-storage-driver` | driver | Name of storage driver to use | none |
```

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+| `remote-storage-bucket` | bucket | Bucket name | none |
+| `remote-storage-key` | access key | Optional access key | empty |
```
Comment (shiftedreality) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:42]: Let's move access key before secret key

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md (hunk 1 above)
Comment (hguthrie) [reply src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md:25]: Okay, I will check that. It should not have built successfully if it is broken, so could be a weird thing I did.

**Comment by hguthrie** [comment]
@shiftedreality I changed from draft PR to review. If you are satisfied with my fixes that you requested, then I can merge to the integration branch and move on to the next set of updates. 🙂 OR, you can advise me on further information, but I think this gets the intro work started.

**Comment by hguthrie** [comment]
Merging to integration as discussed in Slack.

**Pull request merged**